### PR TITLE
bump chart version and make raft storage type retain

### PR DESCRIPTION
**Pull request by Macbeth98**

### CODE REVIEW

File: charts/siglensent/templates/worker-deployment.yaml [hunk 1]
```diff
@@ -59,9 +59,6 @@ spec:
           name: cert-volume
         {{- end }}
       volumes:
-      - name: data-volume
-        persistentVolumeClaim:
-          claimName: gcp-pd-pvc
```
Comment (AndrewHess) [inline charts/siglensent/templates/worker-deployment.yaml:64]: So now the the `data-volume` from above is used, is that right?
```
  volumeClaimTemplates:
  - metadata:
      name: data-volume
    spec:
      accessModes: [ "ReadWriteOnce" ]
      storageClassName: gcp-pd-rwo
      resources:
        requests:
          storage: {{ .Values.pvc.size }}
```

File: charts/siglensent/Chart.yaml [hunk 2]
```diff
@@ -7,13 +7,13 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.1
+version: 0.1.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.2.21-ent1"
+appVersion: "0.2.22-ent1"
```
Comment (AndrewHess) [inline charts/siglensent/Chart.yaml:16]: I thought we were on 0.2.35 or something?

File: charts/siglensent/templates/worker-deployment.yaml (hunk 1 above)
Comment (Macbeth98) [reply charts/siglensent/templates/worker-deployment.yaml:64]: Yes, this is redundant.

File: charts/siglensent/Chart.yaml (hunk 2 above)
Comment (Macbeth98) [reply charts/siglensent/Chart.yaml:16]: Yeah I will change and make a push

**Review by AndrewHess — APPROVED** [review]

**Pull request merged**